src/sass/essay/_base.scss

```diff
@@ -0,0 +1,94 @@
+// Copyright (C) 2018 The Trustees of Indiana University
```

Comment:
I believe this should be `(C) 2020` since this a new component, I believe. Let's double-check the requirements laid out for copyright banners here: https://indiana-university.github.io/